sections/featured-product.liquid

```diff
@@ -315,25 +317,28 @@
                                 {{- price_break.minimum_quantity -}}
```

Comment:
Are you ok if I do this in a follow up?

Reply:
Sounds good! not a blocker for this PR. But to clarify, this is not the same as "+ show more".

- For quantities (i.e. "10+"), we need the `+` to be part the translation string and announced by screen readers. This is because "10" and "10+" does not have the same meaning.

- For "+ show more", we actually don't need the `+` to be translated and we want to hide it from screen readers. Because the `+` in this case is purely ornamental (similar to an icon glyph).

Reply:
Good point! I have updated the issue